config/routes.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :inventories, except: [:update, :edit]
 
+  resources :recipes
```

Comment:
Great work with the routes, it's almost perfect. Please modify this so the available routes are only for the actions that we are using. You can decide to exclude the ones we are not going to use (like the routes for inventories below in the code), or you can manually add the routes like this `resources :recipes, only: [:index, :new, :create, (etc.)]`